app/models/UserAnswers.scala

```diff
@@ -118,4 +121,14 @@ final case class UserAnswers(
   }
 }
 
+object UserAnswers {
+
+  def isPublicServicePensionsRemedy(chargeType: ChargeType, request: DataRequest[AnyContent]): Option[Boolean] = chargeType match {
```

Comment:
I think it would be better to have this as a method on the UserAnswers class. There's no need to pass DataRequest into this class. Less is more...